uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/request-modal-details-toil.controller.spec.js

```diff
@@ -47,20 +50,31 @@ define([
 
     beforeEach(inject(function (
       _$componentController_, _$q_, _$log_, _$rootScope_, _AbsenceType_, _AbsenceTypeAPI_,
-      _LeaveRequestInstance_, _TOILRequestInstance_) {
+      _crmAngService_, _LeaveRequestInstance_, _OptionGroup_, _TOILRequestInstance_) {
       $componentController = _$componentController_;
       $log = _$log_;
       $q = _$q_;
       $rootScope = _$rootScope_;
       AbsenceType = _AbsenceType_;
+      crmAngService = _crmAngService_;
       AbsenceTypeAPI = _AbsenceTypeAPI_;
       LeaveRequestInstance = _LeaveRequestInstance_;
+      OptionGroup = _OptionGroup_;
       TOILRequestInstance = _TOILRequestInstance_;
 
       spyOn($log, 'debug');
       spyOn(AbsenceTypeAPI, 'calculateToilExpiryDate').and.callThrough();
       spyOn(AbsenceType, 'canExpire').and.callThrough();
       spyOn(AbsenceType, 'calculateToilExpiryDate').and.callThrough();
+      spyOn(OptionGroup, 'valuesOf').and.callThrough();
+
+      crmAngService.loadForm = function () {
```

Comment:
I think it's better to put this inside the "'when user opens TOIL accrual options group editor" describe block

Reply:
Done.